src/BlsBaseDSS.sol

```diff
@@ -0,0 +1,212 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8;
+
+import "forge-std/console.sol";
+import {IBaseDSS} from "./interfaces/IBaseDSS.sol";
+import {Constants} from "./interfaces/Constants.sol";
+import {BN254} from "./entities/BN254.sol";
+import {ICore} from "./interfaces/ICore.sol";
+import {BaseDSSOperatorLib} from "./entities/BaseDSSOperatorLib.sol";
+import {BlsBaseDSSLib} from "./entities/BlsBaseDSSLib.sol";
+import {IStakeViewer} from "./interfaces/IStakeViewer.sol";
+
+abstract contract BlsBaseDSS is IBaseDSS {
```

Comment:
Import `BaseDSS` here.

Reply:
Need a different state than BaseDSS, that is why have created a parallel to baseDSS for DSS's that want to build on BLS.